images/installer/Dockerfile

```diff
@@ -12,23 +12,54 @@ LABEL name="openshift/origin-ansible" \
       io.k8s.display-name="openshift-ansible" \
       io.k8s.description="A containerized openshift-ansible image to let you run playbooks to install, upgrade, maintain and check an OpenShift cluster" \
       io.openshift.expose-services="" \
-      io.openshift.tags="openshift,install,upgrade,ansible"
+      io.openshift.tags="openshift,install,upgrade,ansible" \
+      vcs-url="https://github.com/openshift/openshift-ansible" \
+      vcs-type="git" \
+      version="alpha"
 
 USER root
 
+# ansible and pip are in EPEL
+RUN yum install -y epel-release && yum clean all -y
+
+# workaround for https://github.com/openshift/openshift-ansible/issues/3111
+# install ansible via pip to lock version
+RUN yum install -y  --setopt=tsflags=nodocs python-pip python-devel && yum clean all -y
+RUN pip install -Iv ansible==2.2.0.0
+
+COPY ./images/installer/bin /usr/bin
+COPY ./images/installer/bin/user_setup /tmp
```

Comment:
Since we are optimizing the image, we could reduce the number of layers by having a single `COPY` instruction, and placing all the files under `./images/installer/root`. Then, we can group together all `RUN` instructions into a single one.